lib/grplatt.gi

```diff
@@ -923,7 +923,8 @@ InstallGlobalFunction(LatticeViaRadical,function(arg)
       HN:=Image(hom,H);
       c:=LatticeByCyclicExtension(f,
 	  [u->IsSubset(HN,u),u->IsSubset(HN,u)])!.conjugacyClassesSubgroups;
-    elif select<>fail and (select=IsPerfectGroup  or select=IsSimpleGroup) then
+    elif select<>fail and (select=IsPerfectGroup  or select=IsNonabelianSimpleGroup)
```

Comment:
`IsNonabelianSimpleGroup` implies `IsPerfectGroup` now, though, so this would be redundant.
```suggestion
    elif select=IsPerfectGroup then
```

Reply:
No, this is not a check, but a parameter check for particular selection functions given to the routine (to only calculate subgroups satisfying some condition). If the user only wants perfect or nonabelian almost simple subgroups, it is sufficient to initialize with perfect subgroups in the radical factor.

Reply:
ahhhh, OK, I see, my bad!